lib/active_support/cache/redis_store.rb

```diff
@@ -147,16 +153,14 @@ def fetch_multi(*names)
           memo
         end
 
-        begin
+        failsafe(:fetch_multi_write) do
```

Comment:
This is for writing data missing in cache store to cache store

According to https://api.rubyonrails.org/classes/ActiveSupport/Cache/Store.html#method-i-fetch_multi
```
Fetches data from the cache, using the given keys. If there is data in the cache with the given keys, then that data is returned. Otherwise, the supplied block is called for each key for which there was no data, and the result will be written to the cache and returned. Therefore, you need to pass a block that returns the data to be written to the cache. If you do not want to write the cache when the cache is not found, use read_multi
```